src/tzdata/build.jl

```diff
@@ -87,7 +87,7 @@ function build(
 
         if !isempty(tz_source_dir)
             @info "Extracting $version tzdata archive"
-            regions = intersect(regions, readarchive(archive))
+            regions = union!(intersect(regions, readdir(artifact_dir)), CUSTOM_REGIONS)
```

Comment:
did you mean to change `archive`->`artifact_dir`?

```suggestion
            regions = union!(intersect(regions, readdir(archive)), CUSTOM_REGIONS)
```

Reply:
```suggestion
            regions = union!(intersect(regions, readarchive(archive)), CUSTOM_REGIONS)
```
Good catch but it should be this instead